docs/_data/nakadi-event-bus-api.yaml

```diff
@@ -122,6 +122,14 @@ paths:
             The flow id of the request, which is written into the logs and passed to called services. Helpful
             for operational troubleshooting and log analysis.
           type: string
+        - name: writers
+          in: query
+          description: |
+            Will query all subscriptions that provided user or service can write to based on authorization section. Service or user
+            should be listed explicitly on authorization section.
+          type: string
```

Comment:
@burimshala do you think it is enough to have a single writer? could we extend it to array like 
`?writer_data_type=type1,type2&writer_data_value=value1,value2`. I also suggest to add readers right away or in the following PR.

Reply:
We already were going in that direction, but together with @a1exsh we decided not to because YAGNI and more problems were popping out.

Reply:
is the intended request gonna happen with single user all the time or you will do several requests for the same subscription?

Reply:
The intended logic is to have only 1 user or service to query subscriptions.